### DISCUSSION TITLE
[Kobra Go] Extruder fan is always on and 100%

### DISCUSSION THREAD

**Discussion by Fallstern** [1coderookie/Klipper4KobraGoNeo#15] [Q&A] [answered]
Hello, can't control extruder's fan, it is always 100% when printing and when printed IDLE. Does anybody have the same issue?

**Comment by 1coderookie** [comment, marked as answer]
Hi,
you're talking about the [part cooling fan](https://1coderookie.github.io/KobraGoNeoInsights/hardware/printhead/#go_6), right?  
Or about the [heatsink fan](https://1coderookie.github.io/KobraGoNeoInsights/hardware/printhead/#heatsink-cooling-fan)?  

Assuming you're talking about the part cooling fan: were you able to control it's speed with the stock firmware?
AC changed the type of fan at the NEO at a certain point, and that one doesn't work with PWM anymore. Many ppl only realize that after switching to Klipper and then trying to alter the speed using the slider. Check the link above and then scroll down to the section of the NEO. Compare you part cooling fan with the ones being shown and let me know which model is being used at your GO. 

Also, please post the fan sections of your `printer.cfg`.

**Reply by Fallstern** [reply]
I don't know what was this, but today this problem just disappeared. Previously it was 100% by default, but now I can control fan using web interface and slicer
![image](https://github.com/1coderookie/Klipper4KobraGoNeo/assets/15169456/ad48d7d5-ffce-4348-966b-ec2ecd2e5475)

**Reply by 1coderookie** [reply]
Ha, nice! Glad you got it sorted :)
I'll close this discussion then..
